### PR TITLE
Introduce cypress integration tests

### PR DESCRIPTION
**Pull request by NipuniBhagya**
```
identity-apps
  |
  |--apps
  |--components
  |--configs
  |--features
  |--modules
  |--scripts
  |--tests
      |
      |--fixtures
      |--integration
      |--output
      |--plugins
      |--output
      |--plugins
      |--scripts
      |--support
      |--cypress.env.json
      |--cypress.json
      |--lerna.json
      |--package.json
      |--tsconfig.json

```

### CODE REVIEW

**Review by brionmario — APPROVED** [review]

File: tests/integration/groups/visit-groups-page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    });
+
+    before(() => {
+        HousekeepingUtils.performCleanUpTasks();
+    });
+
+    it("CDS_1.1 - User login and navigation in the develop section", function () {
+        cy.login(USERNAME, PASSWORD, SERVER_URL, CONSOLE);
+    });
+});
```
Comment (jeradrutnam) [inline tests/integration/groups/visit-groups-page.spec.ts:43]: Missing new-line at EOF

File: tests/modules/cypress-base/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# `cypress-base`
+
+> TODO: description
```
Comment (jeradrutnam) [inline tests/modules/cypress-base/README.md:3]: Please avoid TODOs

File: tests/modules/cypress-base/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+## Usage
+
+```
+const cypressBase = require('cypress-base');
+
+// TODO: DEMONSTRATE API
```
Comment (jeradrutnam) [inline tests/modules/cypress-base/README.md:10]: Please avoid TODOs

File: tests/modules/cypress-base/package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "@wso2is-testing/cypress-base",
+    "version": "0.0.0",
```
Comment (jeradrutnam) [inline tests/modules/cypress-base/package.json:3]: Invalid version

File: tests/fixtures/example.json [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
+
```
Comment (jeradrutnam) [inline tests/fixtures/example.json:6]: Unwanted new line

File: tests/integration/email-templates/email-templates-test-case-constants.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+Object.defineProperty(exports, "__esModule", { value: true });
```
Comment (jeradrutnam) [inline tests/integration/email-templates/email-templates-test-case-constants.js:20]: Missing new line

**Comment by brionmario** [comment]
Merged this PR in-order to perform further refactoring. The new PR https://github.com/wso2/identity-apps/pull/1205 will fix the issues.

File: tests/integration/email-templates/email-templates-test-case-constants.js (hunk 6 above)
Comment (brionmario) [reply tests/integration/email-templates/email-templates-test-case-constants.js:20]: Fixed with https://github.com/wso2/identity-apps/pull/1205.

File: tests/fixtures/example.json (hunk 5 above)
Comment (brionmario) [reply tests/fixtures/example.json:6]: Fixed with https://github.com/wso2/identity-apps/pull/1205

File: tests/modules/cypress-base/package.json (hunk 4 above)
Comment (brionmario) [reply tests/modules/cypress-base/package.json:3]: Fixed with https://github.com/wso2/identity-apps/pull/1205

File: tests/modules/cypress-base/README.md (hunk 3 above)
Comment (brionmario) [reply tests/modules/cypress-base/README.md:10]: Fixed with https://github.com/wso2/identity-apps/pull/1205

File: tests/modules/cypress-base/README.md (hunk 2 above)
Comment (brionmario) [reply tests/modules/cypress-base/README.md:3]: Fixed with https://github.com/wso2/identity-apps/pull/1205

File: tests/integration/groups/visit-groups-page.spec.ts (hunk 1 above)
Comment (brionmario) [reply tests/integration/groups/visit-groups-page.spec.ts:43]: Fixed with https://github.com/wso2/identity-apps/pull/1205

**Pull request merged**